tests/common/fixtures/tacacs.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import pytest
+from pytest_ansible.errors import AnsibleConnectionFailure
+from tests.tacacs.utils import setup_tacacs_client, setup_tacacs_server, load_tacacs_creds,\
+                    cleanup_tacacs, restore_tacacs_servers, print_tacacs_creds
+
+logger = logging.getLogger(__name__)
+TACACS_CREDS_FILE = 'tacacs_creds.yaml'
+
+# Following UT not enable TACACS test:
+
+
+def drop_all_ssh_session(duthost):
+    try:
+        duthost.shell('sudo ss -K sport ssh')
+    except AnsibleConnectionFailure:
```

Comment:
>AnsibleConnectionFailure [](http://example.com/codeflow?start=11&length=24)

Do you want to use something similar to `TestCase.assertRaises`? If no exception thrown, will it fail the test? If yes, you need to fail explicitly.

Reply:
The code here is not a test, we want catch exception and continue without fail UT.

All ansible connection before TACACS enabled will not control by TACACS per-command authorization.

'sudo ss -K sport ssh' will close all SSH connection to device include current connection, so duthost.shell method will throw a AnsibleConnectionFailure exception after this command finish. 

Because all SSH connection dropped, Ansible will create new SSH connection with TACACS enabled.

Reply:
If no exception thrown, should we treat it as passing or failing?

Reply:
Throw exception here is excepted, because we drop current connection.
This part of code not run inside any part of UT, so the throw exception or not doesn't impact any UT result.